.markdownlint.json

```diff
@@ -1,7 +1,11 @@
 {
   "default": true,
-  "MD013": { "line_length": 100 },
+  "MD013": {
+    "line_length": 100,
+    "code_blocks": false,
+    "tables": false
+  },
```

Comment:
@dasconnor I am not familiar with this. Please comment.

Reply:
ahh, this is a good change. These are the settings for the line length check. We used to need to manually ignore code blocks, so this will make it so those are automatically exempt. Also excludes tables. I'm good with this.